### DISCUSSION TITLE
Auswahl Kunden und Projekte

### DISCUSSION THREAD

**Discussion by matttno** [kimai/kimai#2988] [Questions & Answers]
Hallo!
Nach dem Update auf die Version 1.16.6 gibt es offenbar mit der Suche in der Maske Meine Zeiten und Administration-Zeiterfassung ein Problem. Wenn man einen Kunden auswählt wird nicht automatisch nach entsprechenden Projekten vorselektiert. Dh man kann einen Kunden und dazu ein Projekt eines anderen Kunden auswählen. In der Zeiterfassungsmaske funktioniert das wie gehabt.
LG Matthias

**Comment by kevinpapst** [comment]
Du hast vollkommen recht. Hab den Fehler gefunden und korrigiert. Neues Release kommt in Kürze
